packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,12 +10,12 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+-   [*] Fix error when pasting deeply nested structure content [#55613]
 
 ## 1.107.0
 -   [*] Social Icons: Fix visibility of inactive icons when used with block based themes in dark mode [#55398]
 -   [*] Synced Patterns: Fix visibility of heading section when used with block based themes in dark mode [#55399]
 -   [*] Classic block: Add option to convert to blocks [#55461]
--   [*] Fix error when pasting deeply nested structure content [#55613]
```

Comment:
@fluiddot Still not sure how your commit here 887979a55e834065a870a9fdba6e6f1d1bad8c22 resulted in placing the line item in 1.101.7. My only guess was the force push on that branch confused the git history but not enough to cause a conflict 🤷

Reply:
That's interesting, the PR where the change was introduced added the entry to the `Unreleased` section ([reference](https://github.com/WordPress/gutenberg/pull/55613/files#diff-d69739058d3a853b3b1346be9c94e43f82b7013cf8f8c10d8eea6b0d9e3f98ac)).

> My only guess was the force push on that branch confused the git history but not enough to cause a conflict 🤷

Yep, I understand that GitHub should have warned about the conflict but it didn't. Thanks @jhnstn for fixing it 🙇 !